src/plugins/data/common/datasets/types.ts

```diff
@@ -256,6 +258,29 @@ export interface DatasetField {
   // TODO:  osdFieldType?
 }
 
+export enum DatasetCanvasBannerComponentType {
+  CALLOUT = 'callout',
```

Comment:
thanks for making this a string. but could we follow current pattern could we make this all caps: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/data/common/datasets/types.ts#L136

might be a nitpick but it's easier to write code expecting strings to be in a certain casing